src/plugin/plugin.rs

```diff
@@ -88,15 +104,21 @@ where
                 )
                     .chain()
                     .in_set(RapierTransformPropagateSet),
+                systems::on_add_entity_with_parent,
+                systems::on_change_world,
+                apply_deferred,
```

Comment:
I don't believe apply_deferred are needed anymore, ever since https://github.com/bevyengine/bevy/pull/9822 has been merged into bevy main (0.13 i think).

Reply:
I believe you're right, I'd love some tests for that behaviour 🤔, 1 frame delays are difficult to track down.

Reply:
@Aceeri I'm curious about your input concerning 1 frame delays and particular tests (manual or automated) that could be performed to be confident with this change.

Reply:
> @Aceeri I'm curious about your input concerning 1 frame delays and particular tests (manual or automated) that could be performed to be confident with this change.

I believe this is fine, the apply_deferred here as to make sure any newly created colliders/rigid body handles were added to the entity before we processed them in the later functions, but bevy#9822 should automatically add that apply_deferred for us now.

I may have added a test for this but I forget, if we don't it'd just be a test that manually steps the app once and checks that something in the later functions is correctly updated.